### PR TITLE
BulletSubject component (new)

### PR DESCRIPTION
**Pull request by fermarinsanchez**

### CODE REVIEW

**Review by MIGUELez11 — CHANGES_REQUESTED** [review]
Please review the comments, then, i'll re-review it 😋.

PS: I think the component is not correctly exported. Does `pacakges/components/src/misc/index.js` have an export for `BulletSubject`?

File: packages/components/src/misc/BulletSubject/BulletSubject.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { BulletSubjectStyles } from './BulletSubject.styles';
+import { BULLET_SUBJECT_DEFAULT_PROPS, BULLET_SUBJECT_PROP_TYPES } from './BulletSubject.constants';
+import { ImageLoader } from '../ImageLoader';
+
+const BulletSubject = ({ color, icon, size }) => {
+  const { classes } = BulletSubjectStyles({ size });
+  const iconToShow = !!icon ? icon : null;
+  const handleSize = size === 'lg' ? 16 : 12;
+
+  console.log(iconToShow);
```
[the remark this answers is not in the document]
Comment (fermarinsanchez) [reply packages/components/src/misc/BulletSubject/BulletSubject.js]: removed!

File: packages/components/src/misc/BulletSubject/BulletSubject.styles.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const getIconSize = (theme, size) => {
+  return {
+    lg: { height: theme.spacing['4'] },
+    sm: { height: theme.spacing['3'] },
+  }[size];
+};
+
+export const BulletSubjectStyles = createStyles((theme, { size }) => {
+  console.log(theme.other);
```
[the remark this answers is not in the document]
Comment (fermarinsanchez) [reply packages/components/src/misc/BulletSubject/BulletSubject.styles.js]: removed!

File: packages/components/src/misc/BulletSubject/BulletSubject.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  return (
+    <Box className={classes.bubble} style={{ backgroundColor: color }}>
+      {iconToShow && (
+        <ImageLoader
+          forceImage
+          height={handleSize}
+          imageStyles={handleSize}
+          src={iconToShow}
+          alt="subject icon"
```
[the remark this answers is not in the document]
Comment (fermarinsanchez) [reply packages/components/src/misc/BulletSubject/BulletSubject.js:20]: Totally true. The component now receive a new property for alt text and the default property is "subject icon".

**Review by MIGUELez11 — APPROVED** [review]
Looks right to me 😄

**Pull request merged**